packages/components/inputs/rich-text-input/src/editor.tsx

```diff
@@ -56,23 +106,21 @@ const Editor = (props) => {
         ) {
           toggle();
         }
+        const refObj: TRichtTextEditorBodyRef = {
+          containerRef: ref,
+          registerContentNode,
+        };
         return (
           <Constraints.Horizontal max={props.horizontalConstraint}>
             <Stack scale="xs" alignItems="flex-end">
-              <EditorWrapper
-                isDisabled={props.isDisabled}
-                isReadOnly={props.isReadOnly}
-              >
+              <EditorWrapper isDisabled={props.isDisabled}>
                 <RichTextBody
-                  ref={{
-                    containerRef: ref,
-                    registerContentNode,
-                  }}
+                  ref={refObj as unknown as Ref<TRichtTextEditorBodyRef>}
```

Comment:
I'm sorry for using the assertion but I don’t know how to make it work otherwise

Reply:
Yeah that's fine, this `ref` is kind of a mess anyway.